app/src/main/java/edu/grinnell/kdic/MainActivity.java

```diff
@@ -85,27 +84,19 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         // set the layout to use
         setContentView(R.layout.activity_main);
-
         setupNavigation(); // setup the nav drawer and navigation functionality
```

Comment:
Remove these inline comments as well. Use inline comments when you're doing something that absolutely requires explanation